src/python/pants/help/help_info_extracter_test.py

```diff
@@ -96,6 +97,14 @@ def test_deprecated(self):
     self.assertEqual('do not use this', ohi.removal_hint)
     self.assertIsNotNone(ohi.deprecated_message)
 
+  def test_enum(self):
+    class LogLevel(Enum):
+      INFO = 'info'
+      DEBUG = 'debug'
+    kwargs = {'choices': LogLevel}
```

Comment:
Instead of `{'choices': LogLevel}`, we want `{'type': LogLevel, 'default': LogLevel.INFO}`. That's how we register enums. This is going to require tweaking the implementation to get this test working.

Reply:
My mistake, thanks for that!